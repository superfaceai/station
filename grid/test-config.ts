import { NockConfig } from '@superfaceai/testing';

export const nockConfig: NockConfig = {
  testInstance: expect,
};
