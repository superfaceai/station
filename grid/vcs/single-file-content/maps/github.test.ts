import { SuperfaceTest } from '@superfaceai/testing';

import { buildSuperfaceTest } from '../../../test-config';

describe(`vcs/single-file-content/github`, () => {
  let superface: SuperfaceTest;

  beforeEach(() => {
    superface = buildSuperfaceTest({
      profile: 'vcs/single-file-content',
      provider: 'github',
    });
  });

  describe('SingleFileContent', () => {
    it('should perform successfully', async () => {
      await expect(
        superface.run({
          useCase: 'SingleFileContent',
          input: {
            owner: 'superfaceai',
            repo: 'astexplorer',
            path: 'README.md',
            ref: 'master',
          },
        })
      ).resolves.toMatchSnapshot();
    });

    it('should map error', async () => {
      await expect(
        superface.run({
          useCase: 'SingleFileContent',
          input: {
            owner: 'superfaceai',
            repo: 'astexplorer',
            path: 'README.md',
            ref: 'made-up',
          },
        })
      ).resolves.toMatchSnapshot();
    });
  });
});
