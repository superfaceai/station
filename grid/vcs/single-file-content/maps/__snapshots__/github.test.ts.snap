// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`vcs/single-file-content/github SingleFileContent should map error 1`] = `
Err {
  "error": "MappedHTTPError: Expected HTTP error
Properties: {
  \\"message\\": \\"No commit found for the ref made-up\\",
  \\"description\\": \\"https://docs.github.com/v3/repos/contents/\\",
  \\"statusCode\\": 404
}
AST Path: definitions[0].statements[4].responseHandlers[0].statements[1]
Original Map Location: Line 20, column 7",
}
`;

exports[`vcs/single-file-content/github SingleFileContent should perform successfully 1`] = `
Ok {
  "value": Object {
    "content": "IyMgQVNUIGV4cGxvcmVyCgpbIVtKb2luIHRoZSBjaGF0IGF0IGh0dHBzOi8v
Z2l0dGVyLmltL2FzdGV4cGxvcmVyL0xvYmJ5XShodHRwczovL2JhZGdlcy5n
aXR0ZXIuaW0vYXN0ZXhwbG9yZXIvTG9iYnkuc3ZnKV0oaHR0cHM6Ly9naXR0
ZXIuaW0vYXN0ZXhwbG9yZXIvTG9iYnk/dXRtX3NvdXJjZT1iYWRnZSZ1dG1f
bWVkaXVtPWJhZGdlJnV0bV9jYW1wYWlnbj1wci1iYWRnZSZ1dG1fY29udGVu
dD1iYWRnZSkKWyFbQnVpbGQgU3RhdHVzXShodHRwczovL3RyYXZpcy1jaS5v
cmcvZmtsaW5nL2FzdGV4cGxvcmVyLnN2Zz9icmFuY2g9bWFzdGVyKV0oaHR0
cHM6Ly90cmF2aXMtY2kub3JnL2ZrbGluZy9hc3RleHBsb3JlcikKClBhc3Rl
IG9yIGRyb3AgY29kZSBpbnRvIHRoZSBlZGl0b3IgYW5kIGluc3BlY3QgdGhl
IGdlbmVyYXRlZCBBU1Qgb24gaHR0cHM6Ly9hc3RleHBsb3Jlci5uZXQvCgpU
aGUgQVNUIGV4cGxvcmVyIHByb3ZpZGVzIGZvbGxvd2luZyBjb2RlIHBhcnNl
cnM6CgotIENTUzoKICAtIFtjc3NvbV1bXQogIC0gW2Nzc3RyZWVdW10KICAt
IFtwb3N0Y3NzXVtdICsgW3Bvc3Rjc3Mtc2FmZS1wYXJzZXJdW10gJiBbcG9z
dGNzcy1zY3NzXVtdCiAgLSBbcmV3b3JrXVtdCi0gW0dyYXBoUUxdW10KLSBH
cmFwaHZpejoKICAtIFtyZWRvdF1bXQotIEhhbmRsZWJhcnM6CiAgLSBbZ2xp
bW1lcl1bXQogIC0gW2hhbmRsZWJhcnNdW10KICAtIFtlbWJlci10ZW1wbGF0
ZS1yZWNhc3RdW10KLSBIVE1MOgogIC0gW2h0bWxwYXJzZXIyXVtdCiAgLSBb
cGFyc2U1XVtdCiAgLSBbQGFuZ3VsYXIvY29tcGlsZXJdW10KICAtIFtAYW5n
dWxhci1lc2xpbnQvdGVtcGxhdGUtcGFyc2VyXVtdCiAgLSBbc3ZlbHRlXVtd
CiAgLSBbcG9zdGh0bWxdW10KICAtIFtoeW50YXhdW10KLSBWdWU6CiAgLSBb
dnVlLXRlbXBsYXRlLWNvbXBpbGVyXVtdCiAgLSBbQHZ1ZS9jb21waWxlci1j
b3JlXVtdCiAgLSBbdnVlLWVzbGludC1wYXJzZXJdW10KLSBbSUNVXVtdCi0g
SmF2YQogIC0gW2phdmEtcGFyc2VyXVtdCi0gSmF2YVNjcmlwdDoKICAtIFth
Y29ybl1bXSArIFthY29ybi1qc3hdW10KICAtIFtiYWJlbC1lc2xpbnRdW10K
ICAtIFtiYWJ5bG9uXVtdCiAgLSBbZXNwcmVlXVtdCiAgLSBbZXNmb3JtYXR0
ZXJdW10KICAtIFtlc3ByaW1hXVtdCiAgLSBbZmxvdy1wYXJzZXJdW10KICAt
IFtoZXJtZXMtcGFyc2VyXVtdCiAgLSBbbWVyaXlhaF1bXQogIC0gW3JlY2Fz
dF1bXQogIC0gW3NlYWZveF1bXQogIC0gW3NoaWZ0XVtdCiAgLSBbdHJhY2V1
cl1bXQogIC0gW3R5cGVzY3JpcHRdW10KICAtIFt0eXBlc2NyaXB0LWVzbGlu
dC1wYXJzZXJdW10KICAtIFt1Z2xpZnktanNdW10KLSBKU09OOgogIC0gW0pT
T05dW10KICAtIFtNb21vYV1bXQotIEx1YToKICAtIFtsdWFwYXJzZV1bXQot
IE1hcmtkb3duOgogIC0gW3JlbWFya11bXQotIFtNRFhdW106CiAgLSBbbWR4
aGFzdF1bXQotIE1vbmtleQogIC0gW21vbmtleV1bXQotIFBIUAogIC0gW3Bo
cC1wYXJzZXJdW10KLSBbUHVnXVtdCi0gUmVndWxhciBFeHByZXNzaW9uczoK
ICAtIFtyZWdleHAtdHJlZV1bXQogIC0gW3JlZ2V4cHBdW10KICAtIFtyZWdq
c3BhcnNlcl1bXQotIFByb3RvY29sIEJ1ZmZlcnM6CiAgLSBbcGJraXRdW10K
LSBTY2FsYQogIC0gW1NjYWxhbWV0YV1bXQotIFNvbGlkaXR5OgogIC0gW3Nv
bGlkaXR5LXBhcnNlci1hbnRscl1bXQotIFNRTDoKICAtIFtzcWxpdGUtcGFy
c2VyXVtdCi0gU3VwZXJmYWNlIExhbmd1YWdlOgogIC0gW3N1cGVyZmFjZS1w
YXJzZXJdW10KLSBbV2ViSURMXVtdCi0gWUFNTDoKICAtIFt5YW1sXVtdCiAg
LSBbeWFtbC1hc3QtcGFyc2VyXVtdCgojIyMgRXhwZXJpbWVudGFsIC8gY3Vz
dG9tIHN5bnRheAoKRGVwZW5kaW5nIG9uIHRoZSBwYXJzZXIgc2V0dGluZ3Ms
IGl0IG5vdCBvbmx5IHN1cHBvcnRzIEVTNS9DU1MzIGJ1dCBhbHNvCgotIEVT
NjogW2Fycm93IGZ1bmN0aW9uc10oaHR0cHM6Ly9naXRodWIuY29tL2x1a2Vo
b2Jhbi9lczZmZWF0dXJlcyNhcnJvd3MpLCBbZGVzdHJ1Y3R1cmluZ10oaHR0
cHM6Ly9naXRodWIuY29tL2x1a2Vob2Jhbi9lczZmZWF0dXJlcyNkZXN0cnVj
dHVyaW5nKSwKICBbY2xhc3Nlc10oaHR0cHM6Ly9naXRodWIuY29tL2x1a2Vo
b2Jhbi9lczZmZWF0dXJlcyNjbGFzc2VzKSwgLi4uCi0gRVM3IHByb3Bvc2Fs
czogW2FzeW5jL2F3YWl0XShodHRwczovL2dpdGh1Yi5jb20vbHVrZWhvYmFu
L2VjbWFzY3JpcHQtYXN5bmNhd2FpdCksIFtvYmplY3QgcmVzdCAvIHNwcmVh
ZF0oaHR0cHM6Ly9naXRodWIuY29tL3NlYm1hcmtiYWdlL2VjbWFzY3JpcHQt
cmVzdC1zcHJlYWQpLCAuLi4KLSBbSlNYXShodHRwczovL2ZhY2Vib29rLmdp
dGh1Yi5pby9qc3gvKSwga25vd24gdGhyb3VnaCBbUmVhY3RdKGh0dHBzOi8v
ZmFjZWJvb2suZ2l0aHViLmlvL3JlYWN0LykuCi0gVHlwZWQgSmF2YVNjcmlw
dCAoW0Zsb3ddKGh0dHA6Ly9mbG93dHlwZS5vcmcvKSBhbmQgW1R5cGVTY3Jp
cHRdKGh0dHA6Ly90eXBlc2NyaXB0bGFuZy5vcmcvKSkKLSBbU0FTU10oaHR0
cDovL3Nhc3MtbGFuZy5jb20vKQoKIyMjIFRyYW5zZm9ybXMKClNpbmNlIGZ1
dHVyZSBzeW50YXggaXMgc3VwcG9ydGVkLCB0aGUgQVNUIGV4cGxvcmVyIGlz
IGEgdXNlZnVsIHRvb2wgZm9yCmRldmVsb3BlcnMgd2hvIHdhbnQgdG8gY3Jl
YXRlIEFTVCB0cmFuc2Zvcm1zLiBJbiBmYWN0LCBmb2xsb3dpbmcgdHJhbnNm
b3JtZXJzCmFyZSBpbmNsdWRlZCBzbyB5b3UgY2FuIHByb3RvdHlwZSB5b3Vy
IG93biBwbHVnaW5zOgoKLSBKYXZhU2NyaXB0CiAgLSBbYmFiZWxdW10gKHY1
LCB2NikKICAtIFtFU0xpbnRdW10gKHYxLCB2MiwgdjMpCiAgLSBbanNjb2Rl
c2hpZnRdW10KICAtIFt0c2xpbnRdW10KLSBIVE1MCiAgLSBbcG9zdGh0bWxd
W10KLSBDU1MKICAtIFtwb3N0Y3NzXVtdCi0gTURYCiAgLSBbbWR4XVtdCi0g
UmVndWxhciBFeHByZXNzaW9ucwogIC0gW3JlZ2V4cC10cmVlXVtdCi0gSGFu
ZGxlYmFycwogIC0gW2dsaW1tZXJdW10KCiMjIyBNb3JlIEZlYXR1cmVzCgot
IFNhdmUgYW5kIGZvcmsgY29kZSBzbmlwcGV0cy4gQ29weSB0aGUgVVJMIHRv
IHNoYXJlIHRoZW0uCi0gQ29weWluZyBhbiBBU1Qgb3IgZHJvcHBpbmcgYSBm
aWxlIGNvbnRhaW5pbmcgYW4gQVNUIGludG8gdGhlIHdpbmRvdyB3aWxsCiAg
cGFyc2UgdGhlIEFTVCBhbmQgdXBkYXRlIHRoZSBjb2RlIHVzaW5nIFtlc2Nv
ZGVnZW5dW10uCi0gT3RoZXJ3aXNlLCB0aGUgY29udGVudCBvZiB0ZXh0IGVk
aXRvciB3aWxsIGJlIHJlcGxhY2VkIHdpdGggdGhlIGNvbnRlbnQgb2YgdGhl
CiAgZmlsZSAoaS5lLiB5b3UgY2FuIGRyYWcgYW5kIGRyb3AgSlMgZmlsZXMp
LgotIENob29zZSBiZXR3ZWVuIG11bHRpcGxlIHBhcnNlcnMgYW5kIGNvbmZp
Z3VyZSB0aGVtLgotIHNoaWZ0K2NsaWNrIG9uIGEgbm9kZSBleHBhbmRzIHRo
ZSBmdWxsIHN1YnRyZWUuCi0gSG92ZXJpbmcgb3ZlciBhIG5vZGUgaGlnaGxp
Z2h0cyB0aGUgY29ycmVzcG9uZGluZyB0ZXh0IGluIHRoZSBzb3VyY2UgY29k
ZQotIEVkaXRpbmcgdGhlIHNvdXJjZSBvciBtb3ZpbmcgdGhlIGN1cnNvciBh
cm91bmQgd2lsbCBhdXRvbWF0aWNhbGx5IGhpZ2hsaWdodAogIHRoZSBjb3Jy
ZXNwb25kaW5nIEFTVCBub2RlIChvciBpdHMgYW5jZXN0b3JzIG9mIGl0IGlz
bid0IGV4cGFuZGVkKQotIFlvdSBjYW4gdXNlIGAkbm9kZWAgaW4gdGhlIGNv
bnNvbGUgdG8gcmVmZXIgdG8gdGhlIGxhc3Qgb3BlbmVkL3RvZ2dsZWQgQVNU
CiAgbm9kZS4KClthY29ybi1qc3hdOiBodHRwczovL2dpdGh1Yi5jb20vUlJl
dmVyc2VyL2Fjb3JuLWpzeApbYWNvcm5dOiBodHRwczovL2dpdGh1Yi5jb20v
dGVybmpzL2Fjb3JuCltAYW5ndWxhci9jb21waWxlcl06IGh0dHBzOi8vYW5n
dWxhci5pby8KW0Bhbmd1bGFyLWVzbGludC90ZW1wbGF0ZS1wYXJzZXJdOiBo
dHRwczovL2dpdGh1Yi5jb20vYW5ndWxhci1lc2xpbnQvYW5ndWxhci1lc2xp
bnQvdHJlZS9tYXN0ZXIvcGFja2FnZXMvdGVtcGxhdGUtcGFyc2VyCltiYWJl
bC1lc2xpbnRdOiBodHRwczovL2dpdGh1Yi5jb20vYmFiZWwvYmFiZWwtZXNs
aW50CltiYWJlbF06IGh0dHBzOi8vYmFiZWxqcy5pby9kb2NzL2FkdmFuY2Vk
L3BsdWdpbnMvCltiYWJ5bG9uXTogaHR0cHM6Ly9iYWJlbGpzLmlvLwpbY3Nz
b21dOiBodHRwczovL2dpdGh1Yi5jb20vTlYvQ1NTT00KW2Nzc3RyZWVdOiBo
dHRwczovL2dpdGh1Yi5jb20vY3NzdHJlZS9jc3N0cmVlCltlbWJlci10ZW1w
bGF0ZS1yZWNhc3RdOiBodHRwczovL2dpdGh1Yi5jb20vZW1iZXItdGVtcGxh
dGUtbGludC9lbWJlci10ZW1wbGF0ZS1yZWNhc3QKW2VzY29kZWdlbl06IGh0
dHBzOi8vZ2l0aHViLmNvbS9lc3Rvb2xzL2VzY29kZWdlbgpbZXNsaW50XTog
aHR0cDovL2VzbGludC5vcmcvCltlc3ByZWVdOiBodHRwczovL2dpdGh1Yi5j
b20vZXNsaW50L2VzcHJlZQpbZXNwcmltYV06IGh0dHBzOi8vZ2l0aHViLmNv
bS9qUXVlcnkvZXNwcmltYQpbZmxvdy1wYXJzZXJdOiBodHRwczovL2dpdGh1
Yi5jb20vZmFjZWJvb2svZmxvdy90cmVlL21hc3Rlci9zcmMvcGFyc2VyCltn
cmFwaHFsXTogaHR0cHM6Ly9mYWNlYm9vay5naXRodWIuaW8vZ3JhcGhxbC8K
W2hlcm1lcy1wYXJzZXJdOiBodHRwczovL2dpdGh1Yi5jb20vZmFjZWJvb2sv
aGVybWVzL3RyZWUvbWFzdGVyL3Rvb2xzL2hlcm1lcy1wYXJzZXIvanMvaGVy
bWVzLXBhcnNlcgpbaHRtbHBhcnNlcjJdOiBodHRwczovL2dpdGh1Yi5jb20v
ZmI1NS9odG1scGFyc2VyMgpbanNjb2Rlc2hpZnRdOiBodHRwczovL2dpdGh1
Yi5jb20vZmFjZWJvb2svanNjb2Rlc2hpZnQKW2x1YXBhcnNlXTogaHR0cHM6
Ly9veHljLmdpdGh1Yi5pby9sdWFwYXJzZS8KW21lcml5YWhdOiBodHRwczov
L2dpdGh1Yi5jb20vbWVyaXlhaC9tZXJpeWFoLwpbcGFyc2U1XTogaHR0cHM6
Ly9naXRodWIuY29tL2luaWt1bGluL3BhcnNlNQpbcGJraXRdOiBodHRwczov
L2dpdGh1Yi5jb20vcmlpaWQvcGJraXQKW3Bvc3Rjc3Mtc2FmZS1wYXJzZXJd
OiBodHRwczovL2dpdGh1Yi5jb20vcG9zdGNzcy9wb3N0Y3NzLXNhZmUtcGFy
c2VyCltwb3N0Y3NzLXNjc3NdOiBodHRwczovL2dpdGh1Yi5jb20vcG9zdGNz
cy9wb3N0Y3NzLXNjc3MKW3Bvc3Rjc3NdOiBodHRwczovL2dpdGh1Yi5jb20v
cG9zdGNzcy9wb3N0Y3NzCltwb3N0aHRtbF06IGh0dHBzOi8vZ2l0aHViLmNv
bS9wb3N0aHRtbC9wb3N0aHRtbApbcmVjYXN0XTogaHR0cHM6Ly9naXRodWIu
Y29tL2JlbmphbW4vcmVjYXN0CltzZWFmb3hdOiBodHRwczovL2dpdGh1Yi5j
b20vS0ZsYXNoL3NlYWZveApbcmV3b3JrXTogaHR0cHM6Ly9naXRodWIuY29t
L3Jld29ya2Nzcy9yZXdvcmsKW3NoaWZ0XTogaHR0cHM6Ly9naXRodWIuY29t
L3NoYXBlc2VjdXJpdHkvc2hpZnQtcGFyc2VyLWpzClt0cmFjZXVyXTogaHR0
cHM6Ly9naXRodWIuY29tL2dvb2dsZS90cmFjZXVyLWNvbXBpbGVyClt0eXBl
c2NyaXB0XTogaHR0cHM6Ly9naXRodWIuY29tL01pY3Jvc29mdC9UeXBlU2Ny
aXB0LwpbdHlwZXNjcmlwdC1lc2xpbnQtcGFyc2VyXTogaHR0cHM6Ly9naXRo
dWIuY29tL2VzbGludC90eXBlc2NyaXB0LWVzbGludC1wYXJzZXIvClt0c2xp
bnRdOiBodHRwczovL3BhbGFudGlyLmdpdGh1Yi5pby90c2xpbnQvClt1Z2xp
ZnktanNdOiBodHRwczovL2dpdGh1Yi5jb20vbWlzaG9vL1VnbGlmeUpTMgpb
d2ViaWRsXTogaHR0cHM6Ly9naXRodWIuY29tL2Rhcm9iaW4vd2ViaWRsMi5q
cwpbcmVkb3RdOiBodHRwczovL2dpdGh1Yi5jb20vcmVkb3Rqcy9yZWRvdApb
cmVtYXJrXTogaHR0cHM6Ly9naXRodWIuY29tL3JlbWFya2pzL3JlbWFyawpb
cmVnZXhwLXRyZWVdOiBodHRwczovL2dpdGh1Yi5jb20vRG1pdHJ5U29zaG5p
a292L3JlZ2V4cC10cmVlCltyZWdleHBwXTogaHR0cHM6Ly9naXRodWIuY29t
L215c3RpY2F0ZWEvcmVnZXhwcApbcmVnanNwYXJzZXJdOiBodHRwczovL2dp
dGh1Yi5jb20vanZpZXJlY2svcmVnanNwYXJzZXIKW3BocC1wYXJzZXJdOiBo
dHRwczovL2dpdGh1Yi5jb20vZ2xheXp6bGUvcGhwLXBhcnNlcgpbcHVnXTog
aHR0cHM6Ly9naXRodWIuY29tL3B1Z2pzL3B1ZwpbZ2xpbW1lcl06IGh0dHBz
Oi8vZ2l0aHViLmNvbS9nbGltbWVyanMvZ2xpbW1lci12bQpbaGFuZGxlYmFy
c106IGh0dHA6Ly9oYW5kbGViYXJzanMuY29tLwpbaWN1XTogaHR0cHM6Ly9m
b3JtYXRqcy5pby9kb2NzL2ludGwtbWVzc2FnZWZvcm1hdC1wYXJzZXIvCltq
c29uXTogaHR0cHM6Ly9naXRodWIuY29tL3Z0cnVzaGluL2pzb24tdG8tYXN0
CltNb21vYV06IGh0dHBzOi8vZ2l0aHViLmNvbS9odW1hbndob2NvZGVzL21v
bW9hCltzcWxpdGUtcGFyc2VyXTogaHR0cHM6Ly9naXRodWIuY29tL2NvZGVz
Y2hvb2wvc3FsaXRlLXBhcnNlcgpbeWFtbF06IGh0dHBzOi8vZ2l0aHViLmNv
bS9lZW1lbGkveWFtbApbeWFtbC1hc3QtcGFyc2VyXTogaHR0cHM6Ly9naXRo
dWIuY29tL211bGVzb2Z0LWxhYnMveWFtbC1hc3QtcGFyc2VyCltlc2Zvcm1h
dHRlcl06IGh0dHBzOi8vZ2l0aHViLmNvbS9taWxsZXJtZWRlaXJvcy9lc2Zv
cm1hdHRlci1wYXJzZXIjcmVhZG1lClttZHhdOiBodHRwczovL21keGpzLmNv
bS8KW21keGhhc3RdOiBodHRwczovL21keGpzLmNvbS9hZHZhbmNlZC9hc3Qj
bWR4aGFzdApbbWR4XTogaHR0cHM6Ly9tZHhqcy5jb20vYWR2YW5jZWQvc3lu
Yy1hcGkKW21vbmtleV06IGh0dHBzOi8vZ2l0aHViLmNvbS9nZW5namlhd2Vu
L21vbmtleS1ydXN0CltzY2FsYW1ldGFdOiBodHRwOi8vc2NhbGFtZXRhLm9y
Zy8KW3NvbGlkaXR5LXBhcnNlci1hbnRscl06IGh0dHBzOi8vZ2l0aHViLmNv
bS9mZWRlcmljb2JvbmQvc29saWRpdHktcGFyc2VyLWFudGxyCltzdXBlcmZh
Y2UtcGFyc2VyXTogaHR0cHM6Ly9naXRodWIuY29tL3N1cGVyZmFjZWFpL3N1
cGVyZmFjZS1wYXJzZXIKW3Z1ZS10ZW1wbGF0ZS1jb21waWxlcl06IGh0dHBz
Oi8vZ2l0aHViLmNvbS92dWVqcy92dWUvdHJlZS9kZXYvcGFja2FnZXMvdnVl
LXRlbXBsYXRlLWNvbXBpbGVyCltAdnVlL2NvbXBpbGVyLWNvcmVdOiBodHRw
czovL2dpdGh1Yi5jb20vdnVlanMvdnVlLW5leHQvYmxvYi9tYXN0ZXIvcGFj
a2FnZXMvY29tcGlsZXItY29yZS9SRUFETUUubWQKW3Z1ZS1lc2xpbnQtcGFy
c2VyXTogaHR0cHM6Ly9naXRodWIuY29tL215c3RpY2F0ZWEvdnVlLWVzbGlu
dC1wYXJzZXIKW3N2ZWx0ZV06IGh0dHBzOi8vZ2l0aHViLmNvbS9zdmVsdGVq
cy9zdmVsdGUKW2h5bnRheF06IGh0dHBzOi8vZ2l0aHViLmNvbS9uaWstZ2Fy
bWFzaC9oeW50YXgKW2phdmEtcGFyc2VyXTogaHR0cHM6Ly9naXRodWIuY29t
L2poaXBzdGVyL3ByZXR0aWVyLWphdmEvdHJlZS9tYXN0ZXIvcGFja2FnZXMv
amF2YS1wYXJzZXIKCiMjIyBDb250cmlidXRpb25zCgpJJ20gaGFwcHkgYWJv
dXQgYW55IGZlZWRiYWNrLCBmZWF0dXJlIHJlcXVlc3Qgb3IgUFIgdG8gbWFr
ZSB0aGlzIHRvb2wgYXMgdXNlZnVsCmFzIHBvc3NpYmxlIQoKLS0tCgojIyMj
IEhvdyB0byBhZGQgYSBuZXcgcGFyc2VyCgoxLiBHbyB0byBgd2Vic2l0ZS9g
LgoyLiBJbnN0YWxsIHRoZSBuZXcgcGFyc2VyIGFzIGRlcGVuZGVuY3k6IGB5
YXJuIGFkZCB0aGVQYXJzZXJgIChvciBgbnBtIGluc3RhbGwgLVMgdGhlUGFy
c2VyYCkKMy4gQ29weSBvbmUgb2YgdGhlIGV4aXN0aW5nIGV4YW1wbGVzIGlu
IGBzcmMvcGFyc2Vycy97bGFuZ3VhZ2V9YC4KNC4gQWRqdXN0IHRoZSBjb2Rl
IGFzIG5lY2Vzc2FyeToKCi0gVXBkYXRlIG1ldGFkYXRhLgotIExvYWQgdGhl
IHJpZ2h0IHBhcnNlciAoYGxvYWRQYXJzZXJgKS4KLSBDYWxsIHRoZSByaWdo
dCBwYXJzaW5nIG1ldGhvZCB3aXRoIHRoZSByaWdodC9uZWNlc3Nhcnkgb3B0
aW9ucyBpbiBgcGFyc2VgLgotIEltcGxlbWVudCB0aGUgYG5vZGVUb1Jhbmdl
YCBtZXRob2QgKHRoaXMgaXMgZm9yIGhpZ2hsaWdodGluZykuCi0gSW1wbGVt
ZW50IHRoZSBgZ2V0Tm9kZU5hbWVgIG1ldGhvZCAodGhpcyBpcyBmb3IgcXVp
Y2sgbG9vayB0aHJvdWdoIHRoZSB0cmVlKS4KLSBJbXBsZW1lbnQgYG9wZW5z
QnlEZWZhdWx0YCBtZXRob2QgZm9yIGF1dG8tZXhwYW5zaW9uIG9mIHNwZWNp
ZmljIHByb3BlcnRpZXMuCi0gRGVmaW5lIGBfaWdub3JlZFByb3BlcnRpZXNg
IHNldCBvciBpbXBsZW1lbnQgYGZvckVhY2hQcm9wZXJ0eWAgZ2VuZXJhdG9y
IG1ldGhvZCBmb3IgZmlsdGVyaW5nLgotIFByb3ZpZGUgYSBgcmVuZGVyU2V0
dGluZ3NgIG1ldGhvZCBpZiBhcHBsaWNhYmxlLgoKIyMjIyBIb3cgdG8gYWRk
IGEgbmV3IHRyYW5zZm9ybWVyCgowLiBHbyB0byBgd2Vic2l0ZS9gLgoxLiBJ
bnN0YWxsIHRoZSBuZXcgdHJhbnNmb3JtZXIgYXMgZGVwZW5kZW5jeS4KMS4g
Q29weSBvbmUgb2YgdGhlIGV4aXN0aW5nIGV4YW1wbGVzIGluIGBzcmMvcGFy
c2Vycy97bGFuZ3VhZ2V9L3RyYW5zZm9ybWVyc2AuCjEuIEFkanVzdCB0aGUg
Y29kZSBhcyBuZWNlc3Nhcnk6CgotIFVwZGF0ZSBtZXRhZGF0YSBhbmQgYGRl
ZmF1bHRQYXJzZXJJRGAuCi0gTG9hZCB0aGUgcmlnaHQgdHJhbnNmb3JtZXIg
KGBsb2FkVHJhbnNmb3JtZXJgKS4KLSBDYWxsIHRoZSB0cmFuc2Zvcm1hdGlv
biBtZXRob2QgaW4gYHRyYW5zZm9ybWAuCi0gQ2hhbmdlIHNhbXBsZSB0cmFu
c2Zvcm1hdGlvbiBjb2RlIGluIGBjb2RlRXhhbXBsZS50eHRgLgoKIyMjIyBC
dWlsZCB5b3VyIG93biB2ZXJzaW9uIGZvciBkZXZlbG9wbWVudAoKMS4gQ2xv
bmUgdGhlIHJlcG9zaXRvcnkuCjIuIEdvIHRvIGB3ZWJzaXRlL2AuCjMuIElu
c3RhbGwgYWxsIGRlcGVuZGVuY2llcyB3aXRoIGB5YXJuIGluc3RhbGxgICh5
b3UgY2FuIHJ1biBgbnBtIGluc3RhbGxgIGFzCiAgIHdlbGwpLgoKUnVuIGB5
YXJuIHJ1biBidWlsZGAgZm9yIHRoZSBmaW5hbCBtaW5pbWl6ZWQgdmVyc2lv
bi4KUnVuIGB5YXJuIHJ1biB3YXRjaGAgZm9yIGluY3JlbWVudGFsIGJ1aWxk
cy4KClJ1biBgeWFybiBzdGFydGAgdG8gc3RhcnQgYSBzaW1wbGUgc3RhdGlj
IHdlYnNlcnZlci4K
",
    "encoding": "base64",
    "size": 8967,
  },
}
`;
